.github/workflows/wheels.yml

```diff
@@ -0,0 +1,46 @@
+---
+name: Wheel Builds
+on:
+  push:
+    tags:
+      - '*'
+jobs:
+  build_wheels_aarch64:
+    name: Build wheels on ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest]
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        name: Install Python
+        with:
+          python-version: '3.7'
+      - uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+        with:
+          platforms: all
+      - name: Install cibuildwheel
+        run: |
+          python -m pip install cibuildwheel==2.0.1 twine
```

Comment:
Unfortunately not in this form, IIRC dependabot only works on requirements files and wouldn't be able to update this here. We can look at moving to a requirements file across the jobs in a follow up to do this. In the past I just go and manually update things when we have a reason to and otherwise leave it.

Reply:
Yeah, fairly sure dependabot would work with `python -mpip install -r requirements-deploy.txt`.  Agreed on the follow-up - let's land this, do the 0.18.1 release and then tidy up build details that are entirely hidden from users.